selfapi.js

```diff
@@ -555,13 +594,25 @@ function getHandlerExporter (app) {
   // `app` is an express-like server app.
   return function (method, path, parameters) {
     // Support restify.
-    if (method === 'delete' && ('del' in app)) {
-      method = 'del';
+    if (method === 'del' && ('delete' in app)) {
+      method = 'delete';
     }
     app[method](path, parameters.handler);
   };
 }
 
+// Stringify Objects, leave non-Objects untouched (e.g. Strings).
+function maybeJsonStringify (value) {
+  if (value instanceof Object) {
```

Comment:
Nit: Here I would prefer bail-out style (short exit trap, longer code afterwards with less indentation), e.g.:

```js
if (!(value instanceof Object)) {
  return value;
}
```

Also, I'm not sure I like function parameters to be formatted like multi-line objects. If you agree, could you please tweak the stringify line like so?

```js
return JSON.stringify(value, options.jsonStringifyReplacer,
  options.jsonStringifySpaces);
```

If you really prefer the original formatting, please disregard this last comment.